### ISSUE TITLE
training time on Tesla K80 

### ISSUE THREAD

**Issue by alanbekker** [keithito/tacotron#96]
I'm training tacotron on LJSpeech using the default params (batchsize=32) 
The log of one the 32 first batches looks:
Generated 32 batches of size 32 in 191.780 sec
Step 1       [195.738 sec/step, loss=1.03379, avg_loss=1.03379]
Step 2       [99.907 sec/step, loss=1.02647, avg_loss=1.03013]
Step 3       [69.131 sec/step, loss=1.02560, avg_loss=1.02862]
Step 4       [52.623 sec/step, loss=1.02535, avg_loss=1.02780]
Step 5       [42.735 sec/step, loss=1.03073, avg_loss=1.02839]
Step 6       [36.833 sec/step, loss=1.02742, avg_loss=1.02823]
Step 7       [32.532 sec/step, loss=0.99367, avg_loss=1.02329]
Step 8       [28.787 sec/step, loss=1.01506, avg_loss=1.02226]
Step 9       [25.883 sec/step, loss=1.01928, avg_loss=1.02193]
Step 10      [23.578 sec/step, loss=1.01565, avg_loss=1.02130]
Step 11      [22.060 sec/step, loss=1.00887, avg_loss=1.02017]
Step 12      [20.509 sec/step, loss=1.01405, avg_loss=1.01966]
Step 13      [19.199 sec/step, loss=1.00242, avg_loss=1.01834]
Step 14      [18.275 sec/step, loss=1.00894, avg_loss=1.01766]
Step 15      [17.543 sec/step, loss=1.00402, avg_loss=1.01676]
Step 16      [16.784 sec/step, loss=0.96318, avg_loss=1.01341]
Step 17      [16.237 sec/step, loss=0.99439, avg_loss=1.01229]
Step 18      [15.765 sec/step, loss=0.99295, avg_loss=1.01121]
Step 19      [15.065 sec/step, loss=0.98162, avg_loss=1.00966]
Step 20      [14.407 sec/step, loss=0.97400, avg_loss=1.00787]
Step 21      [14.061 sec/step, loss=0.96488, avg_loss=1.00583]
Step 22      [13.744 sec/step, loss=0.96628, avg_loss=1.00403]
Step 23      [13.268 sec/step, loss=0.96969, avg_loss=1.00254]
Step 24      [12.826 sec/step, loss=0.95686, avg_loss=1.00063]
Step 25      [12.620 sec/step, loss=0.96109, avg_loss=0.99905]
Step 26      [12.214 sec/step, loss=0.84481, avg_loss=0.99312]
Step 27      [11.882 sec/step, loss=0.94871, avg_loss=0.99147]
Step 28      [11.717 sec/step, loss=0.93698, avg_loss=0.98953]
Step 29      [11.555 sec/step, loss=0.93145, avg_loss=0.98752]
Step 30      [11.249 sec/step, loss=0.92533, avg_loss=0.98545]
Step 31      [10.979 sec/step, loss=0.92952, avg_loss=0.98365]
Step 32      [10.733 sec/step, loss=0.90207, avg_loss=0.98110]

Does the running time makes sense? 
It will take me weeks in order to complete the 250k iterations...

Could someone share his training time with me?

Thanks!

**Comment by keithito** [comment]
It takes a couple hundred iterations before things get up to speed, but your numbers still seem bit slow. What GPU and hyperparameters are you using? I get about 1.1 sec/step on a GTX 1080 Ti.

**Comment by bic-user** [comment]
I have k80 and obtain similar numbers. I guess this is kind of expected:
https://www.reddit.com/r/deeplearning/comments/5mc7s6/performance_difference_between_nvidia_k80_and_gtx/

**Comment by EzequielAdrianM** [comment]
GTX 960 (2GB RAM batch size 4 in order to prevent RAM blowing up) : 1.6 sec/step
Using LJSpeech I can see very understandable audio at only 11K Steps.

**Comment by SoulFireMage** [comment]
I'm getting 12 seconds a step on average on my 1080gtx. Makes me wonder if I've somehow managed to not be using it on the GPU after all!
